engines/bastion_katello/app/assets/javascripts/bastion_katello/repositories/details/views/repository-info.html

```diff
@@ -216,7 +216,7 @@ <h4 translate>Content Counts</h4>
         <tr ng-show="repository.content_type == 'yum'">
           <td translate>Packages</td>
           <td class="align-center">
-            <a ng-href="/katello/content_search#/!=&search[subgrid][repo_id]={{ repository.id }}&search[subgrid][type]=repo_packages">
+            <a ui-sref="products.details.repositories.manage-content.packages({productId: product.id, repositoryId: repository.id})">
```

Comment:
notice i changed this to go to the manage packages/puppet modules page and changed it to allow view_products, but hide the remove buttons if the user does not have permissions to edit_products